src/tests/class_tests/openms/source/Tagger_test.cpp

```diff
@@ -0,0 +1,262 @@
+// --------------------------------------------------------------------------
+//                   OpenMS -- Open-Source Mass Spectrometry
+// --------------------------------------------------------------------------
+// Copyright The OpenMS Team -- Eberhard Karls University Tuebingen,
+// ETH Zurich, and Freie Universitaet Berlin 2002-2018.
```

Comment:
```suggestion
// ETH Zurich, and Freie Universitaet Berlin 2002-2019.
```

can you fix that everywhere?

Reply:
do we have a script to do this globally? We can change that to 2020 in a few weeks again anyway.